lib/ansible/modules/database/postgresql/postgresql_user.py

```diff
@@ -207,11 +207,11 @@
     password: NULL
 '''
 
+from distutils.version import StrictVersion
 from hashlib import md5
 import itertools
 import re
 
-from distutils.version import StrictVersion
```

Comment:
Based on @abadger's comment, due to it being `from hashlib import md5` it should go below `import re`